doc/reference_guide/source/conf.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import subprocess
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Fparser Reference Guide'
+copyright = ('2019-2022, R. W. Ford, P. Peterson, A. R. Porter')
+author = 'R. W. Ford, P. Peterson, A. R. Porter'
```

Comment:
Maybe all the authors from the changelog in the reference guide, since the content comes from the code.

Reply:
Done.